tasks/main.yml

```diff
@@ -1,12 +1,12 @@
 ---
 
 - name: install systemd
-  pacman:
+  ansible.builtin.package:
```

Comment:
When using the fully qualified module name, please also update then minimum required Ansible version in the README.